packages/lit-dev-content/samples/articles/lit-cheat-sheet/bind-tag-name/input-or-textfield.ts

```diff
@@ -0,0 +1,35 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { html as staticHTML, StaticValue } from 'lit/static-html.js';
+import { live } from 'lit/directives/live.js';
+
+@customElement('input-or-textfield')
+export class MyElement extends LitElement {
+  // attribute is false because this is a value that can't be serialized to an
+  // HTML attribute
+  @property({ attribute: false }) tagLiteral: StaticValue|null = null;
+  @property() value = '';
+
+  render() {
+    return html`
+      ${
+        // NOTE: the live() directive prevents setting the .value property if
```

Comment:
I would remove `live` here and the comment. As long as the tag doesn't change, the expense is minimal.